### DISCUSSION TITLE
The system installer unable to detect my wifi conection

### DISCUSSION THREAD

**Discussion by fer78** [dat-linux/community#49] [Issues]
The system installer unable to detect my conection, and says This computer does not satisfy some of the recommended requirements for installing DAT Linux 1.0.2 the system is not connected to the internet. Any help? the live USB, working perfectly with the wifi

**Comment by k1consulting** [comment]
Hi, can you please clarify the problem. If the "live" boot is detecting the wifi, then there should be no issue with the installer also connecting.

The install error may be due to not having minimum resources:
    Min. 3GB RAM
    Min. 20GB disk space

**Reply by fer78** [reply]
Yes, I had that problem when installing it virtually to see how the distro was, I had to increase the features of the virtual machine. But in this case it is a laptop with good features.

**Comment by dat-linux** [comment]
Recommend checking you're actually on the internet when in live boot mode. If so, may be a firewall issue with the installer not able to see the remote repos?

**Reply by fer78** [reply]
I tried to install anyway but it stopped at the end saying it couldn't run the apt-get update command. In the terminal the command worked, it was Calamares that rejected the connection.

**Comment by fer78** [comment]
Thanks to all for the help. 
In the end I had to use a network cable, connect to the router and it worked perfectly. 
I was investigating and it is a Calamares problem because it uses several filters to verify the network connection and if one fails it directly discards the connection. It can be easily modified in the /etc/calamares/ files, but impossible inside the ISO image. Greetings

**Reply by dat-linux** [reply]
Thanks for the feedback. It's basically stock Calamares in terms of the settings. What was the change you made?
